src/libfetchers/input-accessor.cc

```diff
@@ -11,6 +12,30 @@ StorePath InputAccessor::fetchToStore(
     PathFilter * filter,
```

Comment:
No, the filter can be null here. Comparing to `defaultPathFilter` seems uglier than using a nullptr to denote that the caller doesn't want filtering. (It could also be an `std::optional` but that doesn't really matter...)